src/modules/navigator/navigator_main.cpp

```diff
@@ -516,9 +519,9 @@ Navigator::start()
 	/* start the task */
 	_navigator_task = px4_task_spawn_cmd("navigator",
 					 SCHED_DEFAULT,
-					 SCHED_PRIORITY_DEFAULT + 20,
+					 SCHED_PRIORITY_MAX -5,
```

Comment:
@mcharleb With the Linux defines SCHED_PRIORITY_DEFAULT + 20 becomes 119 and obviously fails.